admin-dev/themes/new-theme/js/pages/product/product-map.ts

```diff
@@ -162,6 +162,7 @@ export default {
     defaultDescription: '.serp-default-description',
     watchedDescription: '.serp-watched-description',
     watchedMetaUrl: '.serp-watched-url:input',
+    tagFields: 'input.js-taggable-field',
```

Comment:
```suggestion
    tagFields: '.js-taggable-field',
```

At the same time than fixing the other comment, remove the input element selector, so in the future in case it's not an input, it won't break, the JS class seems enough

Reply:
No I did this on purpose because `js-taggable-field` is added on both the div container and the input and here I need to specifically target the input I don't know why the class is duplicated but my guess is it's related to the translatable input and its default integration on the prestashop form theme

Reply:
Ok, not related to the PR, but added a todo